common/src/registers/MachRegister.C

```diff
@@ -1093,8 +1110,99 @@ namespace Dyninst {
+          case 78: return Dyninst::InvalidReg;
+          case 79: return Dyninst::InvalidReg;
+          case 80: return Dyninst::InvalidReg;
+          case 81: return Dyninst::InvalidReg;
+          case 82: return Dyninst::InvalidReg;
+          case 83: return Dyninst::InvalidReg;
+          case 84: return Dyninst::InvalidReg;
+          case 85: return Dyninst::InvalidReg;
+          case 86: return Dyninst::InvalidReg;
+          case 87: return Dyninst::InvalidReg;
+          case 88: return Dyninst::InvalidReg;
+          case 89: return Dyninst::InvalidReg;
+          case 90: return Dyninst::InvalidReg;
+          case 91: return Dyninst::InvalidReg;
+          case 92: return Dyninst::InvalidReg;
+
+/* End of documented registers */
+/* The rest of these are assigned arbitrary values for internal Dyninst use. */
+          case 1024: return Dyninst::x86::ax;
+          case 1025: return Dyninst::x86::al;
+          case 1026: return Dyninst::x86::ch;
+          case 1027: return Dyninst::x86::dx;
+          case 1028: return Dyninst::x86::dl;
+          case 1029: return Dyninst::x86::bh;
+          case 1030: return Dyninst::x86::sp;
+          case 1031: return Dyninst::x86::si;
+          case 1032: return Dyninst::x86::cf;
+          case 1033: return Dyninst::x86::pf;
+          case 1034: return Dyninst::x86::af;
+          case 1035: return Dyninst::x86::zf;
+          case 1036: return Dyninst::x86::tf;
+          case 1037: return Dyninst::x86::df;
+          case 1038: return Dyninst::x86::flagc;
+          case 1039: return Dyninst::x86::nt_;
+          case 1040: return Dyninst::x86::rf;
+          case 1041: return Dyninst::x86::ac;
+          case 1042: return Dyninst::x86::vip;
+          case 1043: return Dyninst::x86::cr0;
+          case 1044: return Dyninst::x86::cr2;
+          case 1045: return Dyninst::x86::cr4;
+          case 1046: return Dyninst::x86::cr6;
+          case 1047: return Dyninst::x86::dr0;
+          case 1048: return Dyninst::x86::dr2;
+          case 1049: return Dyninst::x86::dr4;
+          case 1050: return Dyninst::x86::dr6;
+          case 1051: return Dyninst::x86::ymm0;
+          case 1052: return Dyninst::x86::ymm2;
+          case 1053: return Dyninst::x86::ymm4;
+          case 1054: return Dyninst::x86::ymm6;
+          case 1055: return Dyninst::x86::zmm0;
+          case 1056: return Dyninst::x86::zmm2;
+          case 1057: return Dyninst::x86::zmm4;
+          case 1058: return Dyninst::x86::zmm6;
+          case 1059: return Dyninst::x86::k0;
+          case 1060: return Dyninst::x86::k2;
+          case 1061: return Dyninst::x86::k4;
+          case 1062: return Dyninst::x86::k6;
+          case 1063: return Dyninst::x86::oeax;
+          case 1064: return Dyninst::x86::gsbase;
+          case 1065: return Dyninst::x86::tr1;
+          case 1066: return Dyninst::x86::tr3;
+          case 1067: return Dyninst::x86::tr5;
+          case 1068: return Dyninst::x86::tr7;
```

Comment:
Is there a reason that the odd  cr, dr, ymm, zmm, and k registers are missing, along with the even tr registers?

Reply:
Because the perl script I was using to generate these had a huge bug in it that I didn't notice. Fixed now.